### ISSUE TITLE
[FR] Streaming data to a table in a floating window

### ISSUE THREAD

**Issue by marcelonyc** [OpenBB-finance/OpenBBTerminal#3892]
**What's the problem with not having this feature?**
I was not able to find this functionality so I built a prototype. If the functionality is there, I spent a few hours learning new stuff :)

I wonder if there is interest in building something like this in the terminal. 

The motivation came from wanting to have a window open with order updates from Coinbase. Something to catch my attention when an order event happens while I am doing other things. I subscribe to a Websocket and I want the table to refresh and change colors based on events.

**Describe the solution you would like**
I built a prototype to test this. You can clone this [branch](https://github.com/marcelonyc/OpenBBTerminal/tree/concept/TkTableWindowThread) from my repo if you want to take it for a spin.

- I create a function stream_to_tk_window in helper_funcs.py
- - This function would work like print_rich_table to inherit OpenBB styling
- The function expects 2 parameters
- - Function object (an async function that returns a dict) and Parameters to the function (dict)
- I used the pythclient get_price to test this. The styling is very basic
Commands
-- crypto
-- price --symbol AAVE-USD --tk True
-- price --symbol AAVE-USD

![Screenshot from 2023-01-07 18-36-48](https://user-images.githubusercontent.com/12512887/211174149-48cf2428-dad3-408d-bd35-8e6837c29ff4.png)

**Additional information**
- I start a Thread with the Tk window. Threads are no fun, need to test this more

**Comment by deeleeramone** [comment]
Totally interested in stuff like this. Streaming price dashboards would be excellent!  :) 

Have you poked around this POC PR? It might help with the threading issue. https://github.com/OpenBB-finance/OpenBBTerminal/pull/3748

**Comment by marcelonyc** [comment]
It took me a couple of weeks to research options. I concluded that this cannot be implemented using the current plotting backend (matplotlib) I tried running the code from #3748 , but I can't get it to run. I will follow that PR to see where it goes.  I could build this as I did in my example using Tkinter and Threads, but I am not convinced it is the right path. 

I will pause looking into this for now and see how the plotly option evolves.
